src/components/App.jsx

```diff
@@ -3,13 +3,17 @@ import React from 'react';
 // import './App.css';
 import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 import WelcomePage from "./WelcomePage/WelcomePage";
-// import SignIn from "./components/sign-in";
+import LoginPage from "./LoginPage/LoginPage";
+import PasswordResetPage from "./PasswordResetPage/PasswordResetPage";
+import PasswordSendPage from "./PasswordResetPage/PasswordSendPage";
 
 const App = (props) => {
   return (
     <Router>
-      <Route path="/" component={WelcomePage} />
-      {/* <Route path="/signin" component={SignIn} /> */}
+      <Route exact path="/" component={WelcomePage} />
+      <Route exact path="/login" component={LoginPage} />
+      <Route exact path="/account/begin_password_reset" component={PasswordResetPage}/>
```

Comment:
i think it is better to do path here, but please double check and make sure

Reply:
https://stackoverflow.com/questions/49162311/react-difference-between-route-exact-path-and-route-path